install/kubernetes/cilium/values.yaml

```diff
@@ -980,6 +980,11 @@ hubble:
       # -- Whether to set the security context on the Hubble UI pods.
       enabled: true
 
+    # -- hubble-ui service configuration.
+    service:
+      type: ClusterIP
+
+
```

Comment:
can we add commented value for nodePort as well ? It will help to show that the option is available by checking values.yaml.

```
# -- Some comment
#nodePort: 1234
```

Reply:
Sure, I can do that

Reply:
Thanks, can you help to run below commands to update related docs as well

```
make -C install/kubernetes docs
make -C Documentation update-helm-values
```